tools/gn

```diff
@@ -237,6 +266,9 @@ def setup_goma(args):
         os.getenv('FLUTTER_GOMA_CREATE_XCODE_SYMLINKS', '0') == '1'):
       goma_gn_args['create_xcode_symlinks'] = True
 
+  if goma_gn_args['use_goma'] and args.goma_ensure_start:
+    goma_ensure_start(goma_gn_args['goma_dir'])
```

Comment:
Should this call be gated by an env variable to execute only on local builds? starting goma this way for CI builds will remove all the visibility from the goma recipe module and can potentially cause issues difficult to triage.

Reply:
If we're un-forking the goma recipe module, then this should be gated to be only for local builds, but if we can't un-fork the recipe module, then driving goma from this script is one way we could get rid of it, and avoid blocking clang rolls going forward.

Reply:
I'll send the CL to prepare the goma module to unfork it when there is an arm64 cipd package